src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/VilkårService.kt

```diff
@@ -283,5 +283,9 @@ class VilkårService(
     fun hentBoutgiftVilkår(behandlingId: BehandlingId): List<Vilkår> =
         vilkårRepository
             .findByBehandlingId(behandlingId)
-            .filter { it.type == VilkårType.UTGIFTER_OVERNATTING } // TODO: Skal også hente løpende utgifter
+            .filter {
```

Comment:
Burde man kaste feil hvis typen er noe annet enn de 3 typene som er aktuelle for boutgifter?

Reply:
Fikset!